### ISSUE TITLE
请问batch_x与batch_x_mark有什么区别呢？batch_x_mark的作用是什么？

### ISSUE THREAD

**Issue by greet001-cai** [zhouhaoyi/Informer2020#501]

**Comment by greet001-cai** [comment]
已解决，谢谢

**Comment by HALF111** [comment]
我也有这个疑问，请问您是怎么解决的呢

**Comment by blacksnail789521** [comment]
It's the timestamp information.

https://github.com/zhouhaoyi/Informer2020/blob/0ac81e04d4095ecb97a3a78c7b49c936d8aa9933/data/data_loader.py#L176-L177

**Comment by HALF111** [comment]
> It's the timestamp information.
> 
> https://github.com/zhouhaoyi/Informer2020/blob/0ac81e04d4095ecb97a3a78c7b49c936d8aa9933/data/data_loader.py#L176-L177
> 
> Also, it seems that a lot of people confuse "mark" with "mask". ([#158 (comment)](https://github.com/zhouhaoyi/Informer2020/issues/158#issuecomment-848691957))

Many thanks for your answer, so does that mean `batch_x` is the original sequence info, and `batch_x_mark` is the timestamp information which is extracted from the original sequence?

**Comment by blacksnail789521** [comment]
Yes, it used the [`time_features`](https://github.com/zhouhaoyi/Informer2020/blob/0ac81e04d4095ecb97a3a78c7b49c936d8aa9933/utils/timefeatures.py#L114) to create the timestamp infos.

**Comment by HALF111** [comment]
> Yes, it used the [`time_features`](https://github.com/zhouhaoyi/Informer2020/blob/0ac81e04d4095ecb97a3a78c7b49c936d8aa9933/utils/timefeatures.py#L114) to create the timestamp infos.

OK, I understand what you mean. Thanks again for your answer!
